### PR TITLE
CF Release

### PR DESCRIPTION
**Pull request by CFPBot**
### Changed
- **all components:** Standardized all the usage docs for v4
- **cf-core:** Updated the base list styles to match latest DM Spec
- **cf-core:** Updated the body copy elements to remove margin from the `last-child`
- **all components:** Corrected the variables correct names, locations, and organization

### Removed
- **all components:** Deprecated code
  - Deprecated webfont mixins
  - Extra select element

## Review

- @cfpb/front-end-team-admin

I am a [bot](https://github.com/cfpb/cfpbot). If I did something wrong, blame a human.

kitten gif

### CODE REVIEW

**Comment by Scotchester** [comment]
One problem we have with reviewing these PRs is that there is no way to tell from just this PR if the version bumps should be major/minor/patch.

File: src/cf-core/package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "cf-core",
-  "version": "4.0.1",
+  "version": "4.2.0",
```
Comment (Scotchester) [inline src/cf-core/package.json]: We also still have this issue where it bumps two increments in a single release.

Comment (jimmynotjim) [inline src/cf-core/package.json]: Ah heck. I really wonder if we need to do condensed Changelog entries right before opening a bot PR when there are multiple changes like this.

**Comment by contolini** [comment]
Closing to test a bot fix.

**Pull request closed without merging**